Hai secret no more (#4952)
endless-sky/endless-sky: data/hai/hai reveal 2 start.txt

			`	The six of you run to your ship as quickly as possible. You look back and see the men were not following you; they were running to their own ships.`
				launch
	on complete
		payment 350000
		"hai slave prereq" ++
		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise to look at the background checks of human pilots before hiring next time.`
	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Large Core Pirates" 1
		fleet "Small Core Pirates" 1

mission "Hai Rescue: Prison Role-Players"
	# You get to rescue the Hai from "First Contact: Hai"
	landing
	name "Return the Prison Role-Players"
	description "Return Hai prison role-players to <destination>. Payment is <payment>."
	source "Sunracer"
	destination "Allhome"
	passengers 5
	repeat 3
	to offer
		has "First Contact: Hai: declined"
		"Hai Rescue: Prison Role-Players (offer count)" < 3
		not "event: hai-human treaty signed"
		not "Hai Rescue: Prison Role-Players: offered"
		# No combat rating requirement since there are no NPCs.
		has "leaks response 5: accepted"

> 		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.

Author: I quite liked the idea that if you knew about the Hai, you could discover some of the kidnapped Hai missions before the events broke. So I made the two most distant ones able to be done whenever, so that the player might already have a bit of background to hook them in (and because, being the most distant, they're the least likely to get done otherwise, so letting them be available earlier gives more opportunity for the content to be encountered equally.
Before the main story, they're isolated incidents, after the main story, they're suddenly part of the web of events that tie together.
But I only made 2 of them able to be found at that point, because I wanted the player to still have to go and do at least 1 mission at the time it came up in order to meet the 3 required and progress the story.

# Copyright (c) 2020 Nomadic Volcano
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

########################################################################

#Investigate a theorised smuggling route and rescue Hai. You have to do at least three of these missions.

#NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.

mission "Hai Rescue: Yeertle Family"
	landing
	name "Return the Yeertle family"
	description "Transport the Yeertle family to <destination>. Payment is <payment>."

> 	source Prime

Author: This whole section is Nomadic's work so I shall attend to it as best I am able. However, it is somewhat intended that the player is a bit lost. They are directed to worlds 'near' Hai space at least, and there are relatively few of those. Plus, if they've encountered the Hai prior to this storyline, they can encounter the two most distantly located rescue missions as standalones before specifically being asked to investigate by Sayari.

> 	source Prime

Reviewer: I can understand the intent to be lost, but it just feels like this sort of game doesn't lend itself well to aimless wandering like this requires a little of. I'm not sure what the best way to preserve the idea of making the player feel "lost" without it being annoying would be.

> 	source Prime

Author: Have added a series of hint missions, one for each rescue mission, that will fire off if you're in the general vicinity and guide you towards the main missions.

	destination Icelake
	passengers 5
	repeat 3
	to offer
		"combat rating" > 50
		has "First Contact: Hai: offered"
		not "event: hai-human treaty signed"
		"Hai Rescue: Yeertle Family (offer count)" < 3
		not "Hai Rescue: Yeertle Family: offered"
	on offer
		"Hai Rescue: Yeertle Family (offer count)" ++
		conversation

> 			`You look ahead a few dozen meters and you see a short man with a strange necklace. As he walks past a power transfer unit, the man shimmers and you see faintly-visible, squirrel-like, features. It seems the necklace is a projector of a human hologram concealing a Hai. He is sticking to the shadows, trying to avoid being seen.`

Author: Nope, pls fix.

			`	Do you want to talk to him?`
			choice
				`	(Talk to him.)`
				`	(Come back later.)`
					defer
			`	The man's expression doesn't change much as you approach. His holoprojector may not be sophisticated enough to translate Hai facial expressions. You can see exposed wires on the device; this is probably a makeshift projector he produced with limited resources. A well-made version might keep Hai hidden in human space for a long time.`
			`	From the shaking voice, you can tell he's frightened.`
			`	"C-can I help you?"`
			choice
				`	"You're a Hai. Why are you this far into human space?"`
					goto far
				`	"Do you need a ride home to Hai space?"`
					goto "ride offer"
			label far
			`	He pauses for a second, perhaps pondering if he can trust you with the answer.`
			`	"I'm a holographer from Icelake. I chartered a flight from Icelake to Allhome with my family. They would have gone on vacation while I would have been at a conference. Instead, the human merchants locked us in a cell and took us here."`

> 			`	"My daughter is very small for her age, and was able to sneak through a vent to unlock our cell. The ship was under-crewed, so we overpowered them, but they had already landed. The ship was locked down; we could not take off. Since then, we've been hiding near here."`

Reviewer: ```suggestion
			`	"My daughter is very small for her age, and was able to sneak through a vent to unlock our cell. The ship was under-crewed, so we overpowered them. Unfortunately, they had already landed, and the ship is locked down. I don't know how to disable it so we can take off. We've been hiding near here since then."`
```
Didn't really like the structuring here. Tried to change it so it flowed a little better.

Author: I think I am going to attempt a broader re-write of this paragraph.

> 			`	"Can you take us back to <destination>? I have considerable savings back there, and can pay you well. Say, <payment>?"`

Reviewer: ```suggestion
			`	"Can you take us back to <destination>? I have considerable savings back there and can pay you well. Say, <payment>?"`
```

			choice
				`	"Sure!"`
				`	"No, I can't."`
					decline

> 			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`

Reviewer: ```suggestion
			`	"Thank you. My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
```

				goto leave
			label "ride offer"

> 			`	"Yes! Some crooked human merchants locked us in a cell and took us here. I've been unable to find anyone who has even heard of Hai space, and I'm afraid to approach the Syndicate. You're the first lead I have!"`

Reviewer: ```suggestion
			`	"Yes! Some crooked human merchants locked us in a cell and took us here. I've been unable to find anyone who has even heard of Hai space, and I'm afraid to approach the Syndicate. You're the first lead I have!`
```
[From #community-tools on Discord](http://baxtercommunications.nl/quotation-marks-multi-paragraph-rule/).

> 			`	"I promise I'll pay you well when we return. I can afford <payment>... when I'm back at <destination>."`

Reviewer: ```suggestion
			`	"I promise I'll pay you well when we return. I can afford <payment> once we're back at <destination>."`
```

			`	"My name is Duniip Yeertle, by the way. My wife and three children are nearby. They have holoprojectors as well."`
			label leave
			`	"Follow me back to my house, and we will make our preparations."`
			choice
				`	(Follow him.)`

> 			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a triad of energetic Hai children eating dinner with their mother.`

Reviewer: ```suggestion
			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a trio of energetic Hai children eating dinner with their mother.`
```
I think trio works better here based on [this](https://comparewords.com/triad/trio).

> 			`	Duniip turns off his hologram and addresses his family, "I found us a pilot! He's going to take us all the way home." This was followed by four cheering family members and one baby's gurgle.`

Reviewer: ```suggestion
			`	Duniip turns off his hologram and addresses his family. "I found us a pilot! He's going to take us all the way home." The whole family cheers.`
```
You've characterized the three Hai children above as being self-sufficient enough to eat dinner as a family unit and be, well, `energetic`. If one of them is young enough it can only gurgle, that implies a level of dependence contradictory to how they've been characterized above.

			`	The family was already packed to leave. Duniip explains they move regularly so they're not found. It takes them under two minutes to finish preparations and don their holoprojectors.`
			`	Your walk back to the ship is largely uneventful until some men with strange scanners detect the Yeertle family. One man throws a small grenade with a green light on top. You recognize it as a low-level EMP device. All five holograms turn off, along with all computers within twenty meters. Fortunately, your ship is outside that range.`
			`	The six of you run to your ship as quickly as possible. You look back and see the men were not following you; they were running to their own ships.`
				launch

> 	on complete

Reviewer: The rest of these missions have log entries upon completion. Was there any particular reason this one didn't?

Author: Evidently an oversight on Nomadic's part.

		payment 350000
		"hai slave prereq" ++

> 		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise to look at the background checks of human pilots before hiring next time.`

Reviewer: ```suggestion
		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise run background checks of human pilots before hiring next time.`
```

Author: Took out too many words.

Reviewer: Whoops! Thanks. Suggestion updated.

Reviewer: Whoops! Thanks. Suggestion updated.

			`	"Captain <last>! Please help us. We need to get home to <destination>."`
				goto "actual questions"
			label questions
			`	"Do you want to know anything else, or can we leave now?"`
			label "actual questions"
			choice
				`	"Of course. Let's go."`
					goto go
				`	"How do you know who I am?"`
					goto who
				`	"How did you get here?"`
					goto how
				`	"How much will you pay me?"`
					goto pay
				`	"Sorry, maybe another time."`
					defer
			label who
			`	"You don't remember me? I'm the first Hai you met. You were so confused back then and didn't know who or what we were. It was very cute."`
				goto questions
			label how
			`	"There are prison role-play cruises in Hai space where we can ride in a brig with human security guards. My research group thought it would be a fun experience, so we all chartered one."`
			`	"It was a trick! They were pirates and took us into human space instead."`
			`	"They didn't feed us much, so we all pretended to be too weak to stand. When one of the pirates came in to check on us, we overpowered him. The other pirate crashed the ship and used an escape pod. Now we're stuck here."`
				goto questions
			label pay

> 			`	"Ah, well, we can give you what is left of our travel budget: <payment>. The money is at <destination> so you'll have to fly us there first."`

Reviewer: ```suggestion
			`	"Ah, well, we can give you what is left of our travel budget: <payment>. The money is on <destination>, so you'll have to fly us there first."`
```

				goto questions
			label go
			`	"Excellent! Everyone pile aboard. Let's go back to <destination>!"`
				accept
	on complete

> 		log `Found some gullible Hai that pirates tricked into a "prison role-play cruise," complete with security guards and a brig. They overpowered the pirate crew who crashed the ship and used escape pods. I returned the Hai to <destination>.`

Reviewer: ```suggestion
		log `Found some gullible Hai that pirates tricked into a "prison role-play cruise," complete with security guards and a brig. They overpowered the pirate crew who crashed the ship and used escape pods. Returned the Hai to <destination>.`
```

	# No NPCs for this one since the Hai are presumed dead.

mission "Hai Rescue: Ooonem"
	landing
	name "Return Ooonem"
	description "Transport the Hai you rescued to <destination>. Payment is <payment>."
	source "Stormhold"
	destination "Makerplace"
	passengers 1
	repeat 3
	to offer
		"combat rating" > 50
		"Hai Rescue: Ooonem (offer count)" < 3
		not "event: hai-human treaty signed"
		not "Hai Rescue: Ooonem: offered"
		has "First Contact: Hai: offered"
		has "Hai Leaks Response 6: active"
		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
	on offer
		"Hai Rescue: Ooonem (offer count)" ++
		conversation
			`In the shadows, you see a short, hooded, woman. There is a lot of hair sticking out from her hood. She might be a Hai. Do you want to talk to her?`
			choice
				`	(Talk to her.)`
				`	(Come back later.)`
					defer
			`	As you approach, she pulls a laser rifle out of her cloak, points it at you, and says, "No! I'm not going back to them!"`

> 			`	She sees your confusion and calms down, "You're not one of them. What do you want with me?"`

Reviewer: ```suggestion
			`	She sees your confusion and calms down. "You're not one of them. What do you want with me?"`
```

			choice
				`	"I was wondering why a Hai is on a pirate planet."`
					goto hai
				`	"Not one of whom?"`
					goto who
				`	"Sorry to bother you. I'll leave you to your business."`
					goto decline
			label hai
			`	"I was going to meet a businessman on Shroud to arrange a large-scale import deal for my research project. I asked a human merchant to take me there, but I was tricked. The businessman doesn't really exist; the deal was faked. The merchant locked me in a cell and brought me here instead. When he opened the cell, I managed to grab his rifle, shoot him, and hide. It's amazing how slowly humans move."`
				goto "more questions"
			label who
			`	"There are some men looking for me. I've managed to escape them so far; they're not as quick as a Hai. I think they're friends of the merchant who locked me up."`
				goto "more questions"
			label pay

> 			`	"I have no money but my family on <planet> can pay."`

Reviewer: ```suggestion
			`	"I have no money, but my family on <planet> can pay."`
```

				goto "more questions"
			label "more questions"
			`	Can you take me back to <destination>? My family can pay for my return.`
			choice
				`	"Yes, I'll take you there."`
				`	"How much will you pay me?"`
					goto pay
				`	"Why are you on a pirate planet?"`
					goto hai

> 				`	"I'm not one of who?"`

Reviewer: ```suggestion
				`	"I'm not one of whom?"`
```

Author: And he questions me about the uncommon use of words in speech...

Reviewer: Yeah, "whom" does seem uncommon, but "who" is actually grammatically incorrect here while "whom" is correct. [Grammarly on this subject](https://www.grammarly.com/blog/who-vs-whom-its-not-as-complicated-as-you-might-think/#:~:text=Whom%20should%20be%20used%20to,the%20subject%20of%20a%20sentence.).

					goto who
				`	"Sorry, but you'll have to find another ride."`
					goto decline
			`	Are you sure? Pirates will chase you. I'd rather not die on your ship.`
			choice
				`	"Don't worry. I know how to dodge pirates."`
				`	"You should find a different ride then."`
					decline
			`	"Okay. I guess I should have introduced myself... it's easy to forget social niceties in a place like this. My name is Ooonem, and I'm a researcher from <planet>. Thank you for agreeing to take me home."`
			`	You notice several men nearby looking intently at Ooonem. The two of you run towards your ship and make it to cover before the men start shooting. Ooonem pulls a grenade out of her cloak and throws it at the men. Gas explodes outward and the men drop dead in seconds.`

> 			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades "tabun gas: twenty-meter radius." This was a nerve gas grenade, using one of the older varieties of the gas. Cheap and plentiful in pirate space.`

Reviewer 2: I'd argue that it would make more sense for the nerve gas here to be sarin instead of tabun. Tabun degrades really quickly over time making it impractical for grenade use, and additionally is both less lethal (taking longer to kill) and harder to turn into a gas. Additionally, some flavor text to the effect of the slavers clutching their throats or shaking on the ground may be a bit truer to life in terms of the effects of nerve gas.

> 			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades "sarin gas: twenty-meter radius." This was a nerve gas grenade, using one of the older varieties of the gas. Cheap and plentiful in pirate space.`

Reviewer: ```suggestion
			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades: "Sarin Gas: twenty-meter radius." This was a nerve gas grenade that uses one of the older varieties of the gas. They're cheap and plentiful in pirate space.`
```

		payment 225000
		"hai slave prereq" ++
		dialog `Ooonem is relieved to return to <planet>. She leads you to her family's home where you're treated to a bountiful dinner. The cuisine is entirely human-Hai fusion dishes. The dessert is accompanied by most delicious spice of all: <payment>.`
	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Large Core Pirates" 1
		fleet "Small Core Pirates" 1

mission "Hai Rescue: Meroot and Lucy"
	landing
	name "Return Meroot and Lucy"
	description "Transport the scientists you rescued to <destination>. Payment is <payment>."
	source Canyon
	destination Newhome
	passengers 2
	repeat 3
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		not "event: hai-human treaty signed"
		not "Hai Rescue: Meroot and Lucy: offered"
		"Hai Rescue: Meroot and Lucy (offer count)" < 3
	on offer
		"Hai Rescue: Meroot and Lucy (offer count)" ++
		conversation

> 			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot a woman within, with tufts of hair across her body. It's probably a Hai. Another woman, clearly human, is walking nearby carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel.`

Reviewer 2: Might be better worded as "She's probably a Hai" instead of "It's probably a Hai", to make it more in line with the previous sentence.

> 			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot a woman within, with tufts of hair across her body. She's probably a Hai. Another woman, clearly human, is walking nearby carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel.`

Reviewer: ```suggestion
			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot something hiding inside with tufts of hair across its body. It's probably a Hai. A woman is approaches the alcove carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel.`
```
Is it possible to tell Hai gender from a quick glance? I don't think it's possible with any other aliens in-game.

Author: I think it's implied that Hai attire or mannerisms are pretty distinct. The gender of the Hai is never in question when they appear.

Reviewer: I think what seemed most odd to me is that the player would notice gender before species. Perhaps a reversal of the progression? How about this?
```suggestion
			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot something hiding inside with tufts of hair across its body. It's probably a Hai, and it looks to be a female one. A woman approaches the alcove carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel..`
```

			`	Do you want to investigate?`
			choice
				`	(Investigate.)`
				`	(Come back later.)`
					defer

> 			`	You knock on the steel sheet and a woman within the alcove slides it to the side. As she emerges, she carefully moves her body to block your view of the alcove.`

Reviewer: ```suggestion
			`	You knock on the steel sheet, and a woman within the alcove slides it to the side. As she emerges, she carefully moves her body to block your view of the alcove.`
```

			`	She asks nervously, "Yes, can I help you?"`
			choice
				`	"Why are you hiding a Hai in there?"`
					goto hiding
				`	"Can I help you?"`
					goto help
				`	"No, I was just walking by. Sorry to bother you."`
					decline
			label hiding

> 			`	"Please don't tell anyone! We were both kidnapped, but we managed to escape. Now we're hiding here, hoping the men don't find us again."`

Reviewer: ```suggestion
			`	"Please don't tell anyone! We were both kidnapped, but we managed to escape. Now we're hiding here, hoping the men don't find us again.`
```

			`	"Actually, if you can give us a ride to <planet>, we can pay when we get there. You'll be followed though, so I hope you have a fast ship."`
				goto "come in"
			label help

> 			`	"Yes! Some men kidnapped my friend and I, but we managed to escape. We've been hiding here, hoping they don't find us again. If you can fly us to <planet>, we can pay you when we arrive. I hope you have a fast ship; they'll probably chase us."`

Reviewer: ```suggestion
			`	"Yes! Some men kidnapped my friend and I, but she got us out and we managed to escape. We've been hiding here, hoping they don't find us again. If you can fly us to <planet>, we can pay you when we arrive. I hope you have a fast ship; they'll probably chase us."`
```
Here is where you could establish Meroot's gender.

				goto "come in"
			label "come in"
			`	"Come in, and we'll talk about it."`
			choice
				`	(Enter their alcove.)`
				`	"No, I can't help you get home."`
					decline

> 			`	The alcove is a dead air space between two utility rooms in the basement of a warehouse. The hole in the wall looks like it was recently dug out by a laser cutter. There's a small refrigerator, a two-person cot, an air processor, and a shelf with typical household items. Books are strewn everywhere; the women must be avid readers. It seems they've set up this tiny space as a decent, albeit small, apartment.`

Reviewer: Based on this description, why was the sheet gone when the player walked up? That's like leaving your front door open.

Author: Evidently an oversight.

			`	"I'm Lucy," says the human woman.`

> 			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a tazer. I woke up in the brig of a space ship."`

Reviewer: ```suggestion
			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a taser. I woke up in the brig of a space ship."`
```
Based on [this](https://www.dailywritingtips.com/taser-or-tazer-tazing-or-tasering/#:~:text=In%20trying%20to%20find%20out,Swift's%20Electric%20Rifle.%E2%80%9D&text=I'm%20leaning%20towards%20Taser,tazed%2C%20tazing%20for%20the%20verb.), I _think_ taser should be spelled with an "s". Weird stuff.

			`	"My experience was similar. I was walking from my biolab to my house on Serpens. Some men jumped out from behind a dune and zapped me."`

> 			`	Meroot continues, "They took us both here. I was able to struggle out of my handcuffs and sneak the keys from the guard. We escaped together, and we've been hiding for weeks."`

Reviewer: ```suggestion
			`	Meroot continues, "They took us both here. I was able to struggle out of my handcuffs and sneak the keys from the guard. We escaped together, and we've been hiding for weeks.`
```

> 			`	"Please take us to <destination>"`

Reviewer: ```suggestion
			`	"Please take us to <destination>!"`
```

			choice
				`	"Sure. Let's go."`
				`	"No, you'll have to find another pilot."`
					decline
			`	The two women put on clothing that thoroughly covers their person. They lead you through a series of ducts and alleys back to the building where your ship is parked. When you have fifty meters left to go, someone shoots Meroot with a stun gun. You and Lucy manage to carry her to your ship and lift off. It appears you're being followed into orbit.`
				launch
	on complete
		log `Rescued a pair of researchers: a Hai woman named Meroot and a human woman named Lucy. They were kidnapped by men who brought them to Canyon. Returned them to <destination> where they now live together. They don't know the identities of the kidnappers.`
		payment 315000
		"hai slave prereq" ++

> 		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot, "It's safer here." The two don't know who the men were that kidnapped them.`

Reviewer: ```suggestion
		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot. "It's safer here," she explains. You ask, but the two have know idea who the men that kidnapped them were.`
```

Author: You managed to keep the wrong "know".

Reviewer: This is unacceptable for the Grammar Man  (suggestion updated)

	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Large Core Pirates" 1
		fleet "Small Core Pirates" 1
			

mission "Hai Rescue: Gipeep"
	landing
	name "Return Gipeep"
	description "Transport the Hai you rescued to <destination>. Payment is <payment>."
	source "Maelstrom"
	destination "Greenbloom"
	passengers 1
	to offer
		"combat rating" > 50
		not "event: hai-human treaty signed"
		has "Unwanted Cargo: done"
		has "Hai Leaks Response 6: active"
		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
	on offer
		conversation
			`You walk by some cargo and hear loud thumping coming from a locked crate. You notice a key nearby. Do you want to check it out?`
			choice
				`	(Examine the crate.)`
				`	(Leave.)`
					decline

> 			`	The key has a symbol you don't recognize, but it looks like an angry cat in a circle. As you get closer, the ordinary-looking crate starts seeming a little strange. It appears to be well-insulated against sound and temperature, but has concealed breathing holes. You can still hear loud thumping.`

Reviewer: ```suggestion
			`	The key has a symbol you don't recognize, but it looks like an angry cat in a circle. As you get closer, the ordinary-looking crate starts to look a little strange. It appears to be well-insulated against sound and temperature, but has concealed breathing holes. You can still hear loud thumping.`
```
Loud thumping, but it's noticeably sound-containing? That's a little contradictory.

Author: Another one that I might attempt a broader re-write.

			`	Do you want to try opening the crate?`
			choice
				`	(Open the crate.)`
				`	(Leave.)`
					decline
			`	You put the key in the lock, turn, and the next thump you hear pushes the crate open so hard it falls over. An angry-looking Hai jumps out and starts yelling at you in its own language.`

> 			`	After a few seconds, the Hai calms down and speaks in your own tongue, "They locked me in a box! I was walking home from a store on <destination> and some humans pulled me into an alley. They shoved me in a box and now I'm..." the Hai pauses and looks around, "not in Hai space?"`

Reviewer: ```suggestion
			`	After a few seconds, the Hai calms down and speaks in your own tongue. "They locked me in a box! I was walking home from a store on <planet>, and some humans pulled me into an alley. They shoved me in a box and now I'm..." the Hai pauses and looks around, "not in Hai space?`
```

			`	"Please take me home. I'm a ship architect from <destination>. We should hurry before they come back."`
			choice
				`	"Of course. Just hop in my ship."`
					goto accept
				`	"No, I'm not going that way any time soon."`

> 			`	"I have nowhere else to go! I'm not exactly conspicuous. I'll be caught soon. Please, I can pay you well."`

Reviewer: ```suggestion
			`	"I have nowhere else to go! I'm not exactly inconspicuous. I'll be caught soon. Please, I can pay you well."`
```
Think you mean [inconspicuous](https://www.dictionary.com/browse/inconspicuous).

			choice
				`	"In that case, I'll take you."`
				`	"No. You'll have to find another pilot."`
					decline
			label accept
			`	You can hear men walking down the hall towards you. You and Gipeep run quickly in the other direction. Soon you hear shots fired, and see them hit a nearby wall. They passed quite close to your head.`
			`	Through luck and skill, the two of you make it to your ship and quickly launch. The escape seemed a little too easy though; they may have been avoiding hitting Gipeep.`
			`	It appears ships are following you into orbit.`
				launch
	on complete

> 		log `Rescued a Hai ship architect named Gipeep who was kidnapped near his home on <destination> and transported in a locked cargo crate. The crate's key had a marking on it similar to the Syndicate logo.`

Reviewer: Maybe the player should note when earlier investigating the key that it looks like the Syndicate logo?

Author: This is an oversight from an earlier version of the story, the symbol in fact does *not* look like the Syndicate logo.

	on offer
		"Hai Rescue: Prison Role-Players (offer count)" ++
		conversation
			`As you set your landing vector, you detect a very faint distress call directed at your ship. Your scanners show it appears to come from a remote island. Do you want to check it out?`
			choice
				`	(Follow the distress signal.)`
				`	(Ignore it and come back later.)`
					defer
			`	You put your ship down in a forest clearing and step out to see some Hai with a directional transmitter and a very broken pirate ship.`
			`	"Captain <last>! Please help us. We need to get home to <destination>."`
				goto "actual questions"
			label questions
			`	"Do you want to know anything else, or can we leave now?"`
			label "actual questions"
			choice
				`	"Of course. Let's go."`
					goto go
				`	"How do you know who I am?"`
					goto who
				`	"How did you get here?"`
					goto how
				`	"How much will you pay me?"`
					goto pay
				`	"Sorry, maybe another time."`
					defer
			label who
			`	"You don't remember me? I'm the first Hai you met. You were so confused back then and didn't know who or what we were. It was very cute."`
				goto questions
			label how

> 			`	"There are prison role-play cruises in Hai space where we can ride in a brig with human security guards. My research group thought it would be a fun experience, so we all chartered one."`

Author: Decadence is as decadence does.

		payment 565000
		"hai slave prereq" ++
		dialog `The Hai researchers thank you for rescuing them from their ill-fated "prison role-play cruise" and promise to stick to safer pastimes in the future. They happily give you <payment>.`
	# No NPCs for this one since the Hai are presumed dead.

mission "Hai Rescue: Oonem Hint"
	landing
	name "A (Short) Angel of Death"
	description "There have been some mysterious deaths on <destination>."
	source
		near "Alcyone" 3
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
		not planet "Stormhold"
	destination Stormhold
	to offer
		has "Hai Leaks Response 6: active"
		has "event: slightly lost"
		not "Hai Rescue: Ooonem: offered"
		not "event: hai-human treaty signed"
		random < 30
	on offer
		conversation
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower, less obvious bands, you hear a pair of captains talking about the pirate world of <planet>. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`

> 			`	You listen in carefully as the speaker relays that there's some kind of hunt going on there. All the ships are being searched before departure, and there's a lot of paranoid goons with weapons hot. A merchant has been killed on his own ship, and random people keep getting into firefights with a short, hooded, 'angel of death' in dark alleyways. So far, no one claims to know what's going on.`

Reviewer 3: "A merchant has been killed on his own ship"
Pirates? Killed a merchant? On hsi own ship? Absolutely shocking tha a pirate would do such a thing.

			`	It's really hard to say what this could be, but it might be worth checking out.`
				accept

mission "Hai Rescue: Ooonem"
	landing
	name "Return Ooonem"
	description "Transport the Hai you rescued to <destination>. Payment is <payment>."
	source "Stormhold"
	destination "Makerplace"
	passengers 1

> 	repeat 3

Reviewer 3: ?

	to offer
		"combat rating" > 50
		"Hai Rescue: Ooonem (offer count)" < 3
		not "event: hai-human treaty signed"
		not "Hai Rescue: Ooonem: offered"
		has "First Contact: Hai: offered"
		has "Hai Leaks Response 6: active"
		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
	on offer
		"Hai Rescue: Ooonem (offer count)" ++
		conversation

> 			`In the shadows, you see a short, hooded, woman. There is a lot of hair sticking out from her hood. She might be a Hai. Do you want to talk to her?`

Reviewer 3: Getting straight on with it, then?

					goto decline
			label hai
			`	"I was going to meet a businessman on Shroud to arrange a large-scale import deal for my research project. I asked a human merchant to take me there, but I was tricked. The businessman doesn't really exist; the deal was faked. The merchant locked me in a cell and brought me here instead. When he opened the cell, I managed to grab his rifle, shoot him, and hide. It's amazing how slowly humans move."`
				goto "more questions"
			label who
			`	"There are some men looking for me. I've managed to escape them so far; they're not as quick as a Hai. I think they're friends of the merchant who locked me up."`
				goto "more questions"
			label pay
			`	"I have no money, but my family on <planet> can pay."`
				goto "more questions"
			label "more questions"
			`	Can you take me back to <destination>? My family can pay for my return.`
			choice
				`	"Yes, I'll take you there."`
				`	"How much will you pay me?"`
					goto pay
				`	"Why are you on a pirate planet?"`
					goto hai
				`	"I'm not one of whom?"`
					goto who
				`	"Sorry, but you'll have to find another ride."`
					goto decline

> 			`	Are you sure? Pirates will chase you. I'd rather not die on your ship.`

Reviewer 3: What? She just asked us to do something, we agreed, and now she's pulling out?

			choice
				`	"Don't worry. I know how to dodge pirates."`
				`	"You should find a different ride then."`
					decline
			`	"Okay. I guess I should have introduced myself... it's easy to forget social niceties in a place like this. My name is Ooonem, and I'm a researcher from <planet>. Thank you for agreeing to take me home."`

> 			`	You notice several men nearby looking intently at Ooonem. The two of you run towards your ship and make it to cover before the men start shooting. Ooonem pulls a grenade out of her cloak and throws it at the men. Gas explodes outward and the men drop to the ground clutching their throats, dead in seconds.`

Reviewer 3: As above, this just seems ridiculous.

			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades: "Sarin Gas: twenty-meter radius." This was a nerve gas grenade that uses one of the older varieties of the gas. They're cheap and plentiful in pirate space.`
			`	You run into your ship and immediately take off.`
				launch
			label decline
			`	The woman runs off into the shadows. You doubt you'll ever see her again.`
				decline
	on complete
		log `Rescued a kidnapped Hai researcher, Ooonem, from Stormhold and returned her to Makerplace. She was tricked into leaving Hai space for a fake business deal, and the merchant who agreed to take her there kidnapped her instead. Men have been hunting her since she escaped. She doesn't know the real identities of the kidnappers.`
		payment 225000
		"hai slave prereq" ++

> 		dialog `Ooonem is relieved to return to <planet>. She leads you to her family's home where you're treated to a bountiful dinner. The cuisine is entirely human-Hai fusion dishes. The dessert is accompanied by most delicious spice of all: <payment>.`

Reviewer 3: Why do you assume I'm a filthy capitalist?

		not planet "Canyon"
	destination Canyon
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		has "Hai Leaks Response 6: active"
		has "event: slightly lost"
		not "Hai Rescue: Meroot and Lucy: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is "deep crevasses" on <planet>, it occurs to you that there would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`
				accept

mission "Hai Rescue: Meroot and Lucy"
	landing
	name "Return Meroot and Lucy"
	description "Transport the scientists you rescued to <destination>. Payment is <payment>."
	source Canyon
	destination Newhome
	passengers 2
	repeat 3
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		not "event: hai-human treaty signed"
		not "Hai Rescue: Meroot and Lucy: offered"
		"Hai Rescue: Meroot and Lucy (offer count)" < 3
	on offer
		"Hai Rescue: Meroot and Lucy (offer count)" ++
		conversation

> 			`As you walk through a major street, you notice a small alcove in the side of one building. Your sharp pilot eyes spot something hiding inside with tufts of hair across its body. It's probably a Hai, and it looks to be a female one. A woman approaches the alcove carrying some food. She looks around to see if she's being followed before entering the alcove and covering it with a sheet of steel..`

Reviewer 3: I guess she's blind because she didn't notice us?

					decline
			`	The alcove is a dead air space between two utility rooms in the basement of a warehouse. The hole in the wall looks like it was recently dug out by a laser cutter. There's a small refrigerator, a two-person cot, an air processor, and a shelf with typical household items. Books are strewn everywhere; the women must be avid readers. It seems they've set up this tiny space as a decent, albeit small, apartment.`
			`	The human woman slides the sheet of steel back into place behind you both once you're in and says, "I'm Lucy."`
			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a taser. I woke up in the brig of a space ship."`
			`	"My experience was similar. I was walking from my biolab to my house on Serpens. Some men jumped out from behind a dune and zapped me."`
			`	Meroot continues, "They took us both here. I was able to struggle out of my handcuffs and sneak the keys from the guard. We escaped together, and we've been hiding for weeks. Please take us to <destination>!"`
			choice
				`	"Sure. Let's go."`
				`	"No, you'll have to find another pilot."`
					decline

> 			`	The two women put on clothing that thoroughly covers their person. They lead you through a series of ducts and alleys back to the building where your ship is parked. When you have fifty meters left to go, someone shoots Meroot with a stun gun. You and Lucy manage to carry her to your ship and lift off. It appears you're being followed into orbit.`

Reviewer 3: In that distance, neither I nor Lucy are shot? Even though we're now moving slower than before? The pursuers don't even catch up?

				launch
	on complete
		log `Rescued a pair of researchers: a Hai woman named Meroot and a human woman named Lucy. They were kidnapped by men who brought them to Canyon. Returned them to <destination> where they now live together. They don't know the identities of the kidnappers.`
		payment 315000
		"hai slave prereq" ++

> 		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot. "It's safer here," she explains. You ask, but the two have no idea who the men that kidnapped them were.`

Reviewer 3: "It's safer here,"
lol, no it's not. Did you forget where Meroot was kidnapped from, Lucy?

	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Large Core Pirates" 1
		fleet "Small Core Pirates" 1
			

mission "Hai Rescue: Gipeep Hint"
	landing
	name "Lucrative Transport"
	description "You've heard there's some suspiciously lucrative transport jobs on <destination>."
	source
		near "Nihal" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
		not planet "Maelstrom"
	destination Maelstrom
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		has "Hai Leaks Response 6: active"
		has "event: slightly lost"
		not "Hai Rescue: Gipeep: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation

> 			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious.`

Reviewer 3: Why is this suspicious?

			`	After a few seconds, the Hai calms down and speaks in your tongue. "They locked me in a box! I was walking home from a store on <planet>, and some humans pulled me into an alley. They shoved me in a box and now I'm..." the Hai pauses and looks around, "not in Hai space?`
			`	"Please take me home. I'm a ship architect from <destination>. We should hurry before they come back."`
			choice
				`	"Of course. Just hop in my ship."`
					goto accept
				`	"No, I'm not going that way any time soon."`
			`	"I have nowhere else to go! I'm not exactly inconspicuous. I'll be caught soon. Please, I can pay you well."`
			choice
				`	"In that case, I'll take you."`
				`	"No. You'll have to find another pilot."`
					decline
			label accept
			`	You can hear men walking down the hall towards you. You both run quickly in the other direction. Soon you hear shots fired, and see them hit a nearby wall. They passed quite close to your head.`
			`	Through luck and skill, the two of you make it to your ship and quickly launch. The escape seemed a little too easy though; they may have been taking pains to avoid hitting your new passenger.`

> 			`	It appears ships are following you into orbit.`

Reviewer 3: This is probably the least ridiculous of these escapes under fire, but the fact that this guy was just in a box, with the key nearby, in whatever hallway you were walking through is still ridiculous.

########################################################################

#Investigate a theorized smuggling route and rescue Hai. You have to do at least three of these missions.

#NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.

mission "Hai Rescue: Yeertle Hint"
	landing
	name "Investigate a Were-Squirrel"
	description "A truly outlandish rumor has been floating around that might be a Hai hiding out on <destination>."
	source
		near "Betelgeuse" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
	destination Prime
	to offer
		"combat rating" > 50
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Yeertle Family: offered"
		not "event: hai-human treaty signed"
		random < 50
	on offer
		conversation
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Often this contains useful tidbits, such as which merchants are ticked off at the moment and whether you should expect to wait a while for refueling. Today, though, you overhear a captain telling an acquaintance a remarkable tale.`

> 			`	Apparently, there is a man-beast on <planet>, and when the other captain laughs at the idea of a modern "werewolf", the first one is quick to clarify that it's more like a five-foot tall "were-squirrel". There seems to be some confused discussion about the mythology of such beasts being revealed when caught staring at bright lights until a third captain, listening in the same as you, butts in to inform them that one of them stole that from a popular show a few years back.`

Reviewer 4: `five-foot tall`
Shouldn't metric system be used?

Author: Probably, it just felt clumsy to use it there.

Reviewer 5: `meter and a half` doesn't sound clumsy to me

Author: ```suggestion
			`	Apparently, there is a man-beast on <planet>, and when the other captain laughs at the idea of a modern "werewolf", the first one is quick to clarify that it's more like a meter and a half tall "were-squirrel". There seems to be some confused discussion about the mythology of such beasts being revealed when caught staring at bright lights until a third captain, listening in the same as you, butts in to inform them that one of them stole that from a popular show a few years back.`
```

#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

########################################################################

#Investigate a theorized smuggling route and rescue Hai. You have to do at least three of these missions.

#NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.

mission "Hai Rescue: Yeertle Hint"
	landing
	name "Investigate a Were-Squirrel"
	description "A truly outlandish rumor has been floating around that there might be a Hai hiding out on <destination>."
	source
		near "Betelgeuse" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
		not planet "Prime"
	destination Prime
	to offer
		"combat rating" > 50

> 		has "Hai Leaks Response 6: offered"

Author: No this is probably a development artefact, but the idea is definitely that none of these should be able to offer either after this point (to minimise narrative disruption) or after a later point when things are resolved.
To my mind I think they should all be while "active" except for the 2 that are meant to be available any time earlier (if you've already done the stowaway mission) as foreshadowing.

			`	Apparently, there is a man-beast on <planet>, and when the other captain laughs at the idea of a modern "werewolf", the first one is quick to clarify that it's more like a meter and a half tall "were-squirrel". There seems to be some confused discussion about the mythology of such beasts being revealed when caught staring at bright lights until a third captain, listening in the same as you, butts in to inform them that one of them stole that from a popular show a few years back.`
			`	You stop listening on your final approach as the conversation descends into a good-natured argument. Nevertheless, it sounds like a lead on a possible stranded Hai that would be worth following up.`
				accept

mission "Hai Rescue: Yeertle Family"
	landing
	name "Return the Yeertle family"
	description "Transport the Yeertle family to <destination>. Payment is <payment>."
	source Prime
	destination Icelake
	passengers 5
	repeat 3
	to offer
		"combat rating" > 50
		has "First Contact: Hai: offered"
		not "event: hai-human treaty signed"
		"Hai Rescue: Yeertle Family (offer count)" < 3

> 		not "Hai Rescue: Yeertle Family: offered"

Reviewer 6: `repeat 3` and this offer count are redundant with one another. Given what the purpose seems to be, these missions shouldn't have `repeat 3` on them, and should then remove the check for if the mission has offered (as that is redundant with how the game already checks if a mission should offer).

	on offer
		"Hai Rescue: Yeertle Family (offer count)" ++
		conversation

> 			`A few dozen meters ahead, you see a short man with a strange necklace. As he walks past a power transfer unit, the man shimmers and you see faintly-visible, squirrel-like, features. It seems the necklace is a projector of a human hologram concealing a Hai. He is sticking to the shadows, trying to avoid being seen.`

Reviewer 6: This concepts of holograms used by Hai in order to mask their appearance hasn't been introduced prior to this point. Seems like the intro should cover this idea so that it doesn't seem to come out of no where.

Author: This kind of is the intro though? There's also a number of references to hologram usage in the culture missions now and in some of the new station's planet descriptions. So players that have been around Hai space probably won't be too surprised, and for players discovering the Hai for the first time, it'll come out of nowhere one way or another really.

	source "Sunracer"
	destination "Allhome"
	passengers 5
	repeat 3
	to offer
		has "First Contact: Hai: declined"
		"Hai Rescue: Prison Role-Players (offer count)" < 3
		not "event: hai-human treaty signed"
		not "Hai Rescue: Prison Role-Players: offered"
		# No combat rating requirement since there are no NPCs.
		has "Hai Leaks Response 6: active"
		# Some subset of these are restricted so you can't pick up enough of them to finish the quest at the same time as starting it.
	on offer
		"Hai Rescue: Prison Role-Players (offer count)" ++
		conversation
			`As you set your landing vector, you detect a very faint distress call directed at your ship. Your scanners show it appears to come from a remote island. Do you want to check it out?`
			choice
				`	(Follow the distress signal.)`
				`	(Ignore it and come back later.)`
					defer
			`	You put your ship down in a forest clearing and step out to see some Hai with a directional transmitter and a very broken pirate ship.`
			`	"Captain <last>! Please help us. We need to get home to <destination>."`
				goto "actual questions"
			label questions
			`	"Do you want to know anything else, or can we leave now?"`
			label "actual questions"

> 			choice

Reviewer 6: Ref: #5440.

				`	"Of course. Let's go."`
					goto go
				`	"How do you know who I am?"`
					goto who
				`	"How did you get here?"`
					goto how
				`	"How much will you pay me?"`
					goto pay
				`	"Sorry, maybe another time."`

> 					defer

Reviewer 6: Really just leaving them like that. Damn.

mission "Hai Rescue: Meroot and Lucy Hint"
	landing
	name "Deep Crevasses"
	description "There could be kidnappers or kidnapped Hai hiding on <destination>."
	source
		near "Zaurak" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
		not planet "Canyon"
	destination Canyon
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Meroot and Lucy: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is "deep crevasses" on <planet>, it occurs to you that there would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`
				accept

mission "Hai Rescue: Meroot and Lucy"
	landing
	name "Return Meroot and Lucy"
	description "Transport the scientists you rescued to <destination>. Payment is <payment>."
	source Canyon
	destination Newhome
	passengers 2
	repeat 3

> 	to offer

Reviewer 6: This mission is missing a condition to have it only offer if you're in the storyline. If that's intentional, then this needs rewritten as it assumes you're in the storyline, as there's no explanation as to why there's a Hai here and/or why the player suspects that what they spot is a Hai.

Author: Two of these missions can offer as long as you've done the mission where you return the Hai stowaway on your ship. This makes it possible for you to have encountered a small number of Hai who have got stuck and need to get home (once you already know the Hai exist and have helped one of them back home) as foreshadowing for the events of this story.

# Copyright (c) 2020 Nomadic Volcano
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 7: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.
```

		payment 315000
		"hai slave prereq" ++
		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot. "It's safer here," she explains, though you have your doubts. You ask, but the two have no idea who the men that kidnapped them were.`
	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Large Core Pirates" 1
		fleet "Small Core Pirates" 1
			

mission "Hai Rescue: Gipeep Hint"
	landing
	name "Lucrative Transport"
	description "You've heard there's some suspiciously lucrative transport jobs on <destination>."
	source
		near "Nihal" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
		not planet "Maelstrom"
	destination Maelstrom
	to offer
		"combat rating" > 50
		has "Hai Leaks Response 6: active"
		has "event: slightly lost"
		not "Hai Rescue: Gipeep: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation

> 			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious for that particular world <planet> is not known for lucrative transport jobs and it could likely be contraband, or worse...`

Author: ```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious for that particular world. <planet> is not known for lucrative transport jobs and it could likely be contraband, or worse...`
```

########################################################################

#Investigate a theorized smuggling route and rescue Hai. You have to do at least three of these missions.

#NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.

mission "Hai Rescue: Yeertle Hint"
	landing
	name "Investigate a Were-Squirrel"

> 	description "A truly outlandish rumor has been floating around that might be a Hai."

Reviewer: ```suggestion
	description "A truly outlandish rumor has been floating around that might be a Hai hiding out on <destination>."
```
The map will show the destination anyways, so I don't see any reason not to include it in the description (since that seems to be rule of thumb).

	source
		near "Betelgeuse" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
	destination Prime
	to offer
		"combat rating" > 50
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Yeertle Family: offered"
		not "event: hai-human treaty signed"
		random < 50
	on offer
		conversation

> 			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. Often this contains useful tidbits like, which merchants are ticked off at the moment, and whether you should expect to wait a while for refueling. Today though you overhear one captain, obviously having encountered a common acquaintance relaying a very remarkable tale.`

Reviewer: ```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Often this contains useful tidbits, like which merchants are ticked off at the moment and whether you should expect to wait a while for refueling. Today, though, you overhear one captain, obviously having encountered a common acquaintance relaying a very remarkable tale.`
```
OR
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Often this contains useful tidbits, such as which merchants are ticked off at the moment and whether you should expect to wait a while for refueling. Today, though, you overhear a captain telling an acquaintance a remarkable tale.`
```
As per norm, top is purely corrections, bottom is my suggestions for flow.

> 			`	Apparently there is a man-beast on Prime, and when the other captain laughs at the idea of a modern 'werewolf' the first one is quick to clarify that it's more like a five foot tall 'were-squirrel'. There seems to be some confused discussion about the mythology of such beasts being revealed when caught staring at bright lights until a third captain, listening in the same as you, butts in to inform them that one of them stole that from a popular show a few years back.`

Reviewer: ```suggestion
			`	Apparently, there is a man-beast on <planet>, and when the other captain laughs at the idea of a modern "werewolf", the first one is quick to clarify that it's more like a five-foot tall "were-squirrel". There seems to be some confused discussion about the mythology of such beasts being revealed when caught staring at bright lights until a third captain, listening in the same as you, butts in to inform them that one of them stole that from a popular show a few years back.`
```

			choice
				`	(Follow him.)`
			`	You walk down a corridor to a stairway that leads to a brick wall. Duniip walks through the wall; apparently this is another of his holograms. You pass through and see a trio of energetic Hai children eating dinner with their mother.`

> 			`	Duniip turns off his hologram and addresses his family, "I found us a pilot! He's going to take us all the way home."`

Reviewer: ```suggestion
			`	Duniip turns off his hologram and addresses his family. "I found us a pilot! He's going to take us all the way home."`
```

			`	The six of you run to your ship as quickly as possible. You look back and see the men were not following you; they were running to their own ships.`
				launch
	on complete
		log `Found a Hai family that had been locked up by some crooked merchants meant to be providing transport. A lively bunch, returned them to <destination>.`
		payment 350000
		"hai slave prereq" ++
		dialog `The Yeertle family is ecstatic to be home after their long ordeal and gladly give you <payment>. They promise to run background checks of human pilots before hiring next time.`
	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Small Core Pirates" 1

mission "Hai Rescue: Prison RP Hint"
	landing
	name "Phantom Distress Calls"

> 	description "Some merchants have mentioned picking up a distress call they couldn't locate the source of."

Reviewer: ```suggestion
	description "Some merchants have mentioned picking up a distress call near <destination> that they couldn't locate the source of."
```

	source
		near "Mirfak" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
	destination Sunracer
	to offer
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Prison Role-Players: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation

> 			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. Out here on the edge of things, towards the core, it is a popular pastime for merchants to swap ghost stories. In all likelihood the majority of these stories can be put down to minor failures from cheap, mass-produced Syndicate equipment. Nevertheless, something about one of them stands out this time.`

Reviewer: ```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Out here on the edge of things and toward the core, it is a popular pastime for merchants to swap ghost stories. In all likelihood, the majority of these stories can be put down to minor failures from cheap, mass-produced Syndicate equipment. Nevertheless, one of them stands out this time.`
```

> 			`	It seems a handful of merchants have all been picking up a very faint distress signal in the vicinity of Sunracer, but every time they've gone to look for it, they've either experienced some kind of engine warning, or a large group of pirates has jumped into the system and they've bugged out.`

Reviewer: ```suggestion
			`	It seems a handful of merchants have all been picking up a very faint distress signal in the vicinity of <planet>, but every time they've gone to look for it, they've either experienced some kind of engine warning, or a large group of pirates has jumped into the system, and they've bugged out.`
```

> 			`	It seems like a long-shot but you wonder if perhaps this might be a legitimate call for help, maybe from someone who can't risk broadcasting themselves too aggressively...`

Reviewer: ```suggestion
			`	It seems like a long-shot, but you wonder if perhaps this might be a legitimate call for help, maybe from someone who can't risk broadcasting themselves too aggressively...`
```

	# No NPCs for this one since the Hai are presumed dead.

mission "Hai Rescue: Oonem Hint"
	landing
	name "A (Short) Angel of Death"

> 	description "There have been some mysterious deaths on Stormhold."

Reviewer: ```suggestion
	description "There have been some mysterious deaths on <destination>."
```

	source
		near "Alcyone" 3
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
	destination Stormhold
	to offer
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Ooonem: offered"
		not "event: hai-human treaty signed"
		random < 30
	on offer
		conversation

> 			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower less obvious bands, you hear a pair of captains talking about the pirate world of Stormhold. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`

Reviewer: ```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower, less obvious bands, you hear a pair of captains talking about the pirate world of <planet>. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`
```
OR
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. To your surprise, on one of the lower, less obvious frequencies, you hear a pair of captains talking about the pirate world of <planet>. Apparently these two have enough connections to profit off the occasional trip there, but one of them is warning the other not to go by at the moment.`
```
Is the use of `band` and not `frequency` a scientifically accurate thing? I'm just more used to hearing `frequency` used in this case, so I thought I'd point it out. Bottom has that edit; top is just grammar.

Author: My understanding is that in the case where frequencies close to each other are capable of being broadcast and can interfere with each other on the same system, the term band is used for each of the selection of frequencies which are spaced apart enough to all be used safely without interference.

> 			`	You listen in carefully as the speaker relays that there's some kind of hunt going on there. All the ships are being searched before departure and there's a lot of paranoid goons with weapons hot. A merchant has been killed on his own ship and random people keep getting into firefights with a short, hooded, 'angel of death' in dark alleyways. So far, no one claims to know what's going on.`

Reviewer: ```suggestion
			`	You listen in carefully as the speaker relays that there's some kind of hunt going on there. All the ships are being searched before departure, and there's a lot of paranoid goons with weapons hot. A merchant has been killed on his own ship, and random people keep getting into firefights with a short, hooded, 'angel of death' in dark alleyways. So far, no one claims to know what's going on.`
```

		fleet "Small Core Pirates" 1

mission "Hai Rescue: Meroot and Lucy Hint"
	landing
	name "Deep Crevasses"

> 	description "There could be kidnappers or kidnapped Hai hiding on Canyon."

Reviewer: ```suggestion
	description "There could be kidnappers or kidnapped Hai hiding on <destination>."
```

	source
		near "Zaurak" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
	destination Canyon
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Meroot and Lucy: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation

> 			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is 'deep crevasses' on <destination> and it occurs to you that it would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`

Reviewer: ```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. You don't hear anything specific to Hai, but when the punchline of a joke is "deep crevasses" on <planet>, it occurs to you that there would be the perfect place for both illicit activities and escapees to hide out under the radar for a long time. It's probably worth a look...`
```
`Canyon` rather than `Canyon in the soandso system` (can't remember the name of it rn) seems to fit better to me.

		log `Rescued a pair of researchers: a Hai woman named Meroot and a human woman named Lucy. They were kidnapped by men who brought them to Canyon. Returned them to <destination> where they now live together. They don't know the identities of the kidnappers.`
		payment 315000
		"hai slave prereq" ++
		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot. "It's safer here," she explains. You ask, but the two have no idea who the men that kidnapped them were.`
	npc
		government "Bounty Hunter that Won't Enter Hai Space"
		personality plunders disables unconstrained
		fleet "Large Core Pirates" 1
		fleet "Small Core Pirates" 1
			

mission "Hai Rescue: Gipeep Hint"
	landing
	name "Lucrative Transport"

> 	description "You've heard there's some suspiciously lucrative transport jobs on Maelstrom."

Reviewer: ```suggestion
	description "You've heard there's some suspiciously lucrative transport jobs on <destination>."
```

	source
		near "Nihal" 4
		government "Republic" "Syndicate"
		attributes "spaceport"
		not attributes "station"
	destination Maelstrom
	to offer
		"combat rating" > 50
		has "Unwanted Cargo: done"
		has "Hai Leaks Response 6: offered"
		has "event: slightly lost"
		not "Hai Rescue: Gipeep: offered"
		not "event: hai-human treaty signed"
		random < 80
	on offer
		conversation

> 			`As you come in to land at the spaceport you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <destination>. Of course it could mean nothing, but it's also very suspicious.`

Reviewer: ```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious.`
```
